src/index.js

```diff
@@ -96,6 +96,51 @@ class RetryChunkLoadPlugin {
           var script = loadScript(jsonpScriptSrc(chunkId), ${maxRetries});
         `;
 
+          const scriptWithoutRetry = `
```

Comment:
Instead of having a separate script for with and without retry, I think we can get the same effect by setting `maxRetries` to 0 further up if the chunk doesn't match

Reply:
Thank you for the quick reply and suggestions. I have added a new test.

I agree that we can achieve the same with maxRetries = 0 but I deliberately separated the code so that when we don't need retry we can exclude the 10+ lines of code related to retry function and cache bust.
This will keep the bundled chunks clean, that do not need retry.

Reply:
OK that makes sense. I just had a quick look at the webpack api and the first argument passed to the hook will actually be the original JSONP template, so you can return that instead of `scriptWithoutRetry`